harness/resizableArrayBufferUtils.js

```diff
@@ -149,3 +149,10 @@ function TestIterationAndResize(iterable, expected, rab, resizeAfter, newByteLen
   }
   assert(resized, "TestIterationAndResize: resize condition should have been hit");
 }
+
+function MayNeedBigInt(ta, n) {
```

Comment:
Thanks for catching this, totally overlooked it.

Reply:
I also completely removed `WriteToTypedArray` since by defining `MayNeedBigInt`,  `WriteToTypedArray(ta, i, j)` is reduced to `ta[i] = MayNeedBigInt(ta, j)`.